core/src/main/resources/application.yml

```diff
@@ -213,3 +213,6 @@ logging:
 
   # Logging format for console
   pattern.console: '%clr([%d{${LOG_DATEFORMAT_PATTERN:HH:mm:ss}}]){faint} %clr(${LOG_LEVEL_PATTERN:%3p}) %clr(%-26.26logger{25}){cyan} %clr(:){faint} %m%n${LOG_EXCEPTION_CONVERSION_WORD:%wEx}'
+  
+server:
```

Comment:
Join server: port: on the same line, i.e server.port: 8080

Reply:
Add a comment using # explaining what is this configuration property used for (web server port), and when should users change it (multiple instances, prevent port collision)